lightning/src/ln/msgs.rs

```diff
@@ -407,6 +429,8 @@ pub struct CommitmentSigned {
 	pub signature: Signature,
 	/// Signatures on the HTLC transactions
 	pub htlc_signatures: Vec<Signature>,
+	/// The partial Taproot signature on the commitment transaction
+	pub partial_signature_with_nonce: Option<PartialSignatureWithNonce>,
```

Comment:
can be called `commitment_partial_signature_with_nonce` even if more verbose than spec